allennlp/models/biaffine_dependency_parser_multilang.py

```diff
@@ -0,0 +1,238 @@
+    use_mst_decoding_for_validation : ``bool``, optional (default = True).
+        Whether to use Edmond's algorithm to find the optimal minimum spanning tree during validation.
+        If false, decoding is greedy.
+    langs_for_early_stop : ``List[str]``, optional, (default = ["en"])
+        Which languages to include in the averaged metrics
+        (that could be used for early stopping).
+    dropout : ``float``, optional, (default = 0.0)
+        The variational dropout applied to the output of the encoder and MLP layers.
+    input_dropout : ``float``, optional, (default = 0.0)
+        The dropout applied to the embedded text input.
+    initializer : ``InitializerApplicator``, optional (default=``InitializerApplicator()``)
+        Used to initialize the model parameters.
+    regularizer : ``RegularizerApplicator``, optional (default=``None``)
+        If provided, will be used to calculate the regularization penalty during training.
+    """
+
+    def __init__(self,
+                 vocab: Vocabulary,
+                 text_field_embedder: TextFieldEmbedder,
+                 encoder: Seq2SeqEncoder,
+                 tag_representation_dim: int,
+                 arc_representation_dim: int,
+                 tag_feedforward: FeedForward = None,
+                 arc_feedforward: FeedForward = None,
+                 pos_tag_embedding: Embedding = None,
+                 use_mst_decoding_for_validation: bool = True,
+                 langs_for_early_stop: List[str] = ["en"],
+                 dropout: float = 0.0,
+                 input_dropout: float = 0.0,
+                 initializer: InitializerApplicator = InitializerApplicator(),
+                 regularizer: Optional[RegularizerApplicator] = None) -> None:
+        super(BiaffineDependencyParserMultiLang, self).__init__(
+                vocab, text_field_embedder, encoder, tag_representation_dim,
+                arc_representation_dim, tag_feedforward, arc_feedforward,
+                pos_tag_embedding, use_mst_decoding_for_validation, dropout,
+                input_dropout, initializer, regularizer)
+
+        self.langs_for_early_stop = langs_for_early_stop
+
+        self._attachment_scores = defaultdict(lambda: AttachmentScores())
+
+    @overrides
+    def forward(
+            self,  # type: ignore
+            words: Dict[str, torch.LongTensor],
+            pos_tags: torch.LongTensor,
+            metadata: List[Dict[str, Any]],
+            head_tags: torch.LongTensor = None,
+            head_indices: torch.LongTensor = None) -> Dict[str, torch.Tensor]:
+        # pylint: disable=arguments-differ
+        """
+        Embedding each language by the corresponding parameters for
+        ``TextFieldEmbedder``. Batches should contain only samples from a
+        single language.
+
+        Parameters
+        ----------
+        words : Dict[str, torch.LongTensor], required
```

Comment:
If these are just duplicating what's in the base class, I'd recommend trimming this down and pointing to the base class docstring.  No need to repeat what's there.